pkg/models/metrics/metricsruleconst.go

```diff
@@ -516,27 +516,44 @@ var RulePromQLTmplMap = MetricMap{
+	"namespace_statefulset_count_used":          `count(kube_statefulset_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_daemonset_count_used":            `count(kube_daemonset_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_jobs_batch_count_used":           `count(kube_job_info{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_cronjobs_batch_count_used":       `count(kube_cronjob_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_pod_count_used":                  `count(kube_pod_created{namespace="$1"}) by (namespace) * on (namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_service_count_used":              `count(kube_service_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_ingresses_extensions_count_used": `count(kube_ingress_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_pvc_count_used":                  `count(kube_persistentvolumeclaim_info{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_secret_count_used":               `count(kube_secret_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_configmap_count_used":            `count(kube_configmap_created{namespace="$1"}) by (namespace) * on(namespace) group_left(label_kubesphere_io_workspace) kube_namespace_labels`,
+	"namespace_cpu_limit_used":                  `round(namespace:container_cpu_usage_seconds_total:sum_rate{namespace!="", namespace="$1"} * on(namespace) group_left(label_kubesphere_io_workspace)(kube_namespace_labels), 0.001)`,
+	"namespace_cpu_request_used":                `round(namespace:container_cpu_usage_seconds_total:sum_rate{namespace!="", namespace="$1"} * on(namespace) group_left(label_kubesphere_io_workspace)(kube_namespace_labels), 0.001)`,
+	"namespace_memory_limit_used":               `namespace:container_memory_usage_bytes_wo_cache:sum{namespace!="", namespace=~"$1"}* on(namespace) group_left(label_kubesphere_io_workspace)(kube_namespace_labels)`,
+	"namespace_memory_request_used":             `namespace:container_memory_usage_bytes_wo_cache:sum{namespace!="", namespace=~"$1"}* on(namespace) group_left(label_kubesphere_io_workspace)(kube_namespace_labels)`,
+	// TODO: metrics to be enabled in ks 3.0
```

Comment:
should remove commented code here, you can keep these code changes in your own repo and submit a pr in feature